### PR TITLE
kevin/getting_started_yen-servers

### PR DESCRIPTION
**Pull request by kevintran128**

### CODE REVIEW

**Review by natalya-patrikeeva — CHANGES_REQUESTED** [review]
I have a few general comments. I think this page needs a lot more editing. 

Titles - consistent capitalization  (make every word capital for every ## level of header)

I think the section on how to connect should be at the top. The information about yen slurm also seems to cover most of the page but this is supposed to be “introduction to the yens” page so we can actually cut a lot of it. 

I would also move “Why use the yen servers” closer to the top and combine it with “introduction section”.  We can link to User Guide/Storage page here too. 

I think we can severely cut the slurm section as it is covered comprehensively - [here](https://rcpedia-dev.stanford.edu/_user_guide/slurm/)

I also don’t see the hardware description for yen1-5 nodes similar to yen-slurm nodes. We should probably also link to user limits page and link to slurm page but skip the detailed description of the slurm nodes.

**Comment by kevintran128** [comment]
Thank you for all your comments @natalya-patrikeeva . I revised accordingly. It should be good to go.

File: docs/_getting_started/yen-servers.md [hunk 1]
```diff
@@ -60,27 +47,29 @@ There are various ways to connect to the `yen` servers.
...
-
-**Long running jobs**
-
-Even when your laptop is capable of doing the job, you may still want to offload that work to the external server.  The server can free up resources for your laptop to use for other tasks such as browsing web sites, reading PDF files, working with spreadsheets, and so forth. If your laptop crashes, it's very convenient for your compute jobs to continue!
-
-**Storage**
-
-The project files and any large output should live on ZFS file system (not in your home). The ZFS capacity is nearly 1 PB (petabyte).
+## Overview of the Yen Computing Infrastructure
+![](/assets/images/yen-computing-infrastructure.png)
```
Comment (natalya-patrikeeva) [inline docs/_getting_started/yen-servers.md:58]: Let's add an alt name for the image

File: docs/_getting_started/yen-servers.md [hunk 2]
```diff
@@ -60,27 +47,29 @@ There are various ways to connect to the `yen` servers.
 * A terminal on [JupyterLab](/_getting_started/jupyter){:target="_blank"}
 * RStudio or Jupyter Notebook on [JupyterLab](/_getting_started/jupyter){:target="_blank"}
 
-When you SSH in to `yen.stanford.edu`, a load-balancer will assign you to `yen1`, `yen2`, `yen3`, `yen4` or `yen5`.  The `yen11`, `yen12`, `yen13`, `yen14`, `yen15`, `yen16`, `yen17`, `yen18`, `yen-gpu1`, `yen-gpu2`, and `yen-gpu3` servers can only be accessed using the [Slurm scheduler.](/_user_guide/slurm){:target="_blank"}
+When you SSH in to `yen.stanford.edu`, a load-balancer will assign you to one of the interactive Yen server: `yen1`, `yen2`, `yen3`, `yen4` or `yen5`.  
+
+The `yen11`, `yen12`, `yen13`, `yen14`, `yen15`, `yen16`, `yen17`, `yen18`, `yen-gpu1`, `yen-gpu2`, and `yen-gpu3` servers can only be accessed using the [Slurm scheduler.](/_user_guide/slurm){:target="_blank"}
 
 !!! tip
     Any work running on an interactive server (`yen[1-5]`) can only be started or stopped from that server.
```
Comment (natalya-patrikeeva) [inline docs/_getting_started/yen-servers.md:55]: bold face only for clarity

**Comment by kevintran128** [comment]
Resolved all comments in the 2nd batch @natalya-patrikeeva

**Review by natalya-patrikeeva — APPROVED** [review]
looks good

**Pull request merged**